pkg/local/sync.go

```diff
@@ -102,7 +102,10 @@ func (s *SyncManager) Sync(rootPath string, remote *uri.URI, changeSet <-chan *C
 		return err
 	}
 	_, err := fileutil.PruneEmptyDirectories(rootPath)
-	return err
+	if err != nil {
+		return err
+	}
+	return WriteActiveOperation(rootPath, "")
```

Comment:
The sync module should be independent from the index file - It's a standalone and we plan to use it also for fs commands.
Anyway - why do we need to write anything here?

Reply:
We said we want to remove the marker from the index file every time we complete a sync action so I thought that the best way to ensure that is to remove it when we finish the sync.
I can move it outside if you think it shouldn't be here

Reply:
Yes, but we need to remove the marker when we make another operation like commit/checkout/pull --force

Reply:
When you perform checkout or commit or pull you write the index anyway so it will be removed